google/cloud/sql/connector/connector.py

```diff
@@ -17,6 +17,54 @@
 import googleapiclient
 
 
+def get_metadata(service, proj_name, inst_name):
```

Comment:
in general it is a good idea not to abbreviate. `project`, `instance` would also better match other interfaces we have: https://github.com/googleapis/google-cloud-python/blob/e6898be297e4938cce02c00d221eaf5d88002162/core/google/cloud/client.py#L199